### ISSUE TITLE
RadarViewer errors on bigint datatype

### ISSUE THREAD

**Issue by tcarolan** [datagrok-ai/public#1962]
Load or convert to a bigint and try to add a RadarViewer. I receive a dart JS exception pasted below. Could be a missing implementation within the big_int_column type. 

NoSuchMethodError: method not found: 'dD' (w.dD is not a function)
	../packages/ddt/src/data_frame/column_types/big_int_column.dart 181:40  qy.a2
	../packages/ddt/src/data_frame/column.dart 429:7                        dJ6.
	../packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 1516:16    Object.fX
	../packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 1489:11    dlq
	../packages/$sdk/lib/js/dart2js/js_dart2js.dart 667:13                  <fn>
	../js/api/js-api-common.js?270 11393:28                                 get min
	webpack://charts/src/viewers/radar/radar-viewer.ts? 47:28               RadarViewer.init
	webpack://charts/src/viewers/radar/radar-viewer.ts? 73:14               RadarViewer.onTableAttached
	../js/api/js-api-common.js?270 16781:14                                 RadarViewer.onFrameAttached

**Comment by ahadzhanian** [comment]
dev 1.16.0
data: [earthquakes.csv](https://dev.datagrok.ai/f/Demo.Files/geo/earthquakes.csv)
To reproduce, add a Radar chart
An error occurs:
```
logging.dart:82 Error: Cannot convert a BigInt value to a number
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 167379:24   detectValue
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 167369:21   doGuessOrdinal
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 167273:10   guessOrdinal
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 165488:101  prepareSeriesDataSchema
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 99019:96    createSeriesDataSimply
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 106122:89   RadarSeriesModel.getInitialData
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 175768:21   SeriesModel.init
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 106114:27   RadarSeriesModel.init
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 174426:28   GlobalModel.<fn>
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 226318:13   each
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 174353:69   GlobalModel.visitComponent
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 181824:18   entity.topologicalTravel
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 174332:59   GlobalModel._mergeOption
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 174822:15   initBase
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 174244:9    GlobalModel._resetOption
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 174217:10   GlobalModel.setOption
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 157727:17   ECharts.setOption
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 215309:20   RadarViewer.render
	api/packages/published/files/Charts/1.0.22/8XKUAzpplPquXQN1ToDoNUeoTvk4LFsf/12/dist/package.js 215130:14   RadarViewer.onTableAttached
	js/api/js-api-common.js?90992 17686:14                                                                     RadarViewer.onFrameAttached
	packages/d4/src/js_viewer_host/js_viewer_host_core.dart 55:7                                               kn.aeM
	packages/d4/src/js_viewer_host/js_viewer_host_core.dart 71:7                                               <fn>
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 213:19                                         e3h.a
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 237:5                                          e3h.
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 187:19                                         dnw.
	packages/$sdk/lib/async/zone.dart 1371:54                                                                  dkg.Ab
	packages/$sdk/lib/async/future_impl.dart 129:12                                                            dgw.
	packages/$sdk/lib/async/future_impl.dart 665:13                                                            Object.xL
	packages/$sdk/lib/async/future_impl.dart 468:7                                                             bu.kc
	packages/$sdk/lib/async/future_impl.dart 51:5                                                              a22.eY
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 164:3                                          Object.n
	packages/d4/src/js_viewer_host/js_viewer_host_meta.dart 37:55                                              <fn>
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 213:19                                         e3h.a
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 237:5                                          e3h.
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 187:19                                         dnw.
	packages/$sdk/lib/async/zone.dart 1371:54                                                                  dkg.Ab
	packages/$sdk/lib/async/future_impl.dart 129:12                                                            dgw.
	packages/$sdk/lib/async/future_impl.dart 665:13                                                            Object.xL
	packages/$sdk/lib/async/future_impl.dart 468:7                                                             bu.kc
	packages/$sdk/lib/async/future_impl.dart 51:5                                                              a22.eY
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 164:3                                          Object.n
	packages/ddt/src/func/func_call.dart 660:39                                                                <fn>
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 213:19                                         e3h.a
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 237:5                                          e3h.
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 187:19                                         dnw.
	packages/$sdk/lib/async/zone.dart 1371:54                                                                  dkg.Ab
	packages/$sdk/lib/async/future_impl.dart 129:12                                                            dgw.
	packages/$sdk/lib/async/future_impl.dart 665:13                                                            Object.xL
	packages/$sdk/lib/async/future_impl.dart 468:7                                                             bu.kc
	packages/$sdk/lib/async/future_impl.dart 51:5                                                              a22.eY
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 164:3                                          Object.n
	packages/ddt/src/func/func_call.dart 343:18                                                                <fn>
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 213:19                                         e3h.a
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 237:5                                          e3h.
	packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 187:19                                         dnw.
	packages/$sdk/lib/async/zone.dart 1371:54                                                                  dkg.Ab
	packages/$sdk/lib/async/future_impl.dart 129:12                                                            dgw.
	packages/$sdk/lib/async/future_impl.dart 665:13                                                            Object.xL
	packages/$sdk/lib/async/future_impl.dart 468:7                                                             bu.kc
	packages/$sdk/lib/async/future_impl.dart 51:5                                                              a22.eY
	
ar_ @ logging.dart:82
cz @ login.dart.js_1.part.js?90992:98134
dart.dJv.$2 @ login.dart.js_1.part.js?90992:111340
h0 @ js_helper.dart:1521
dart.do1 @ js_helper.dart:1489
(anonymous) @ js_dart2js.dart:667
(anonymous) @ dg.ts:51
6log.js:61 [ECharts] `resize` should not be called during main process
```

**Comment by ahadzhanian** [comment]
This issue is also fixed in Charts 1.0.25.
We continue to work on improving the radar chart.
